www/redirects.yaml

```diff
@@ -126,6 +126,8 @@
   toPath: /docs/data-fetching/
 - fromPath: /docs/centralizing-your-sites-navigation/
   toPath: /docs/creating-dynamic-navigation/
+- fromPath: /newsletter/
+  toPath: /contributing/community/#newsletter
```

Comment:
@AishaBlake I'm not sure what the right page for this is, but I'm pretty sure hashes don't work in redirects?

Reply:
@AishaBlake can you please review so I can change the redirect url.